pkg/defaultcomponents/defaults.go

```diff
@@ -68,10 +72,21 @@ import (
 	"go.uber.org/multierr"
 )
 
+var fileLogReceiverFeatureGate = featuregate.GlobalRegistry().MustRegister("adot.filelog.enabled",
+	featuregate.StageAlpha,
+	featuregate.WithRegisterDescription("Enables the File Log Receiver"))
```

Comment:
```suggestion
	featuregate.WithRegisterDescription("Enables the File Log Receiver"))
```
nit: Enabling this feature gate does not `enable` the file log receiver. Enabling the feature gate allows the user to configure and start a collector with the file log receiver. Can we find better wording for this and the other feature gates?

Reply:
How about "Allows for the ADOT Collector to be configured and started with the File Log Receiver"?